### PR TITLE
Update documentation and remove unesessary configuration

### PR DESCRIPTION
**Pull request by mrproliu**
Features of Zipkin v3 have been almost developed, so this pull request is intended for updating the documentation and removing unnecessary configuration items.

### CODE REVIEW

File: zipkin-server/README.md [hunk 1]
```diff
@@ -1,15 +1,14 @@
 # zipkin-server
-Zipkin Server is a Java 1.8+ service, packaged as an executable jar.
+Zipkin Server is a Java 1.11+ service, packaged as an executable jar.
```
Comment (wu-sheng) [inline zipkin-server/README.md]:
```suggestion
Zipkin Server is a Java 11+ service, packaged as an executable jar.
```

File: zipkin-server/README.md [hunk 2]
```diff
@@ -113,98 +111,110 @@ exposition [text format version 0.0.4](https://prometheus.io/docs/instrumenting/
...
-gauge.zipkin_collector.message_bytes.$transport | last count of bytes in a message
+Metric | Tags | Description
+--- | --- | ---
+trace_in_latency_bucket_bucket | sw_backend_instance=$instance_host_$instance_port, protocol=$transport, le=$second_bucket | The process latency histogram of trace data
+trace_in_latency_sum | sw_backend_instance=$instance_host_$instance_port, protocol=$transport | The total process latency of trace data
+trace_analysis_error_count | sw_backend_instance=$instance_host_$instance_port, protocol=$transport | The error number of trace analysis
 
 ## Configuration
-We support ENV variable configuration, such as `STORAGE_TYPE=cassandra3`, as they are familiar to
+We support ENV variable configuration, such as `ZIPKIN_STORAGE=cassandra`, as they are familiar to
```
Comment (wu-sheng) [inline zipkin-server/README.md]:
```suggestion
We support ENV variable configuration, such as `ZIPKIN_STORAGE=cassandra3`, as they are familiar to
```

I think we consistent with previous. Could you rechecck?

File: zipkin-server/README.md [hunk 3]
```diff
@@ -213,223 +223,330 @@ $ java -jar zipkin.jar
...
 ```bash
-$ MEM_MAX_SPANS=1000000 java -Xmx1G -jar zipkin.jar
+$ java -Xmx1G -jar zipkin.jar
 ```
 
 ### Cassandra Storage
-Zipkin's [Cassandra storage component](../zipkin-storage/cassandra) supports Cassandra 3.11.3+
-and applies when `STORAGE_TYPE` is set to `cassandra3`:
+Zipkin's Cassandra storage component supports Cassandra 3.11.3+
+and applies when `ZIPKIN_STORAGE` is set to `cassandra`:
```
Comment (wu-sheng) [inline zipkin-server/README.md]:
```suggestion
and applies when `ZIPKIN_STORAGE` is set to `cassandra3`:
```

Same here.

File: zipkin-server/README.md [hunk 4]
```diff
@@ -213,223 +223,330 @@ $ java -jar zipkin.jar
...
 
-#### Automatic Index Creation
-Zipkin will automatically create new indices as needed. Elasticsearch by default [allows](https://www.elastic.co/guide/en/elasticsearch/reference/current/docs-index_.html#index-creation) automatic creation of said indices, though your local install may have been configured to disallow it. You can verify this in the cluster settings: `action.auto_create_index: false`.
+[BanyanDB](https://github.com/apache/skywalking-banyandb) is a dedicated storage implementation developed by the SkyWalking Team and the community.
+Currently, BanyanDB is still in the PoC stage and it is not recommended to use it in a production environment.
+
+The following apply when `ZIPKIN_STORAGE` is set to `banyandb`:
+    
+    * `ZIPKIN_STORAGE_BANYANDB_HOST`: The host of BanyanDB. Defaults to `127.0.0.1`.
+    * `ZIPKIN_STORAGE_BANYANDB_PORT`: The port of BanyanDB. Defaults to `17912`.
```
Comment (wu-sheng) [inline zipkin-server/README.md]: These should be merged as `targets`. Could you recheck your dependency version of OAP and sync the doc?

File: zipkin-server/README.md [hunk 5]
```diff
@@ -213,223 +223,330 @@ $ java -jar zipkin.jar
...
+
+```bash
+$ ZIPKIN_STORAGE=banyandb java -jar zipkin.jar
+```
 
 ### Legacy (v1) storage components
 The following components are no longer encouraged, but exist to help aid
 transition to supported ones. These are indicated as "v1" as they use
 data layouts based on Zipkin's V1 Thrift model, as opposed to the
 simpler v2 data model currently used.
```
Comment (wu-sheng) [inline zipkin-server/README.md]: Do we need this statement? I think the new of v3 includes making MySQL and PG back to be available.

File: zipkin-server/README.md [hunk 6]
```diff
@@ -213,223 +223,330 @@ $ java -jar zipkin.jar
...
 
-#### MySQL Storage
-Zipkin's [MySQL component](../zipkin-storage/mysql-v1) is tested against MySQL
-5.7 and applies when `STORAGE_TYPE` is set to `mysql`:
+#### MySQL/PostgreSQL Storage
```
Comment (wu-sheng) [inline zipkin-server/README.md]:
```suggestion
### MySQL/PostgreSQL Storage
```

File: zipkin-server/README.md (hunk 5 above)
Comment (mrproliu) [reply zipkin-server/README.md]: Been deleted.

File: zipkin-server/README.md (hunk 4 above)
Comment (mrproliu) [reply zipkin-server/README.md]: The configuration and documentation have been updated.

File: zipkin-server/README.md (hunk 2 above)
Comment (mrproliu) [reply zipkin-server/README.md]: Update the names to `cassandra3`

**Review by wu-sheng — APPROVED** [review]

File: docker/README.md [hunk 7]
```diff
@@ -92,14 +92,13 @@ accordingly.
 
 Ex. If your link name is "storage" for an Elasticsearch container:
 ```
-  ES_HOSTS=http://$STORAGE_PORT_9200_TCP_ADDR:9200
+  ZIPKIN_STORAGE_ES_CLUSTER_NODES=$STORAGE_PORT_9200_TCP_ADDR:9200
```
Comment (jcchavezs) [inline docker/README.md:95]: Can we maintain both variables and deprecate the former? As far as I understand we want to keep retrocompatibility for ease migration.

Comment (wu-sheng) [reply docker/README.md:95]: We could keep some consistent with before. But things like storage selector, it is very skywalking style, can't be aligned with true/false style.

**Pull request merged**